activitysim/abm/models/trip_matrices.py

```diff
@@ -51,6 +51,16 @@ def write_trip_matrices(network_los):
     if "parking_location" in config.setting("models"):
         parking_settings = config.read_model_settings("parking_location_choice.yaml")
         parking_taz_col_name = parking_settings["ALT_DEST_COL_NAME"]
+        auto_nest_name = parking_settings["AUTO_MODE_NEST"]
+
+        # Read trip mode choice settings to get auto modes
+        trip_mode_choice_settings = config.read_model_settings("trip_mode_choice.yaml")
+        trip_mode_choice_nest = config.get_logit_mocel_settings(trip_mode_choice_settings)
+        for alternative in trip_mode_choice_nest["alternatives"]:
+            if alternative["name"] == auto_nest_name:
+                auto_modes = alternative["alternatives"]
+                break
```

Comment:
Suggest you throw an assert statement / runtime error here to make sure the AUTO_MODE_NEST setting exists in the trip mode choice yaml.   Also, I think we want the user to be able to specify None or something to allow them to skip having any auto modes. (There may be some weird transit or walk-only models someone thinks of.)

Reply:
I looked at the configs for prototype_mtc and saw that that has a more complicated nesting structure that the code I wrote wouldn't work for. I went ahead and set it so that the user would need to specify a list in the parking location settings (along with an assertion error saying it's needed).